### ISSUE TITLE
No notifications in a certain situation

### ISSUE THREAD

**Issue by random-solutions** [signalapp/Signal-iOS#5259]
Consider this scenario:
iPhone is offline and a message wants to arrive during this time. Then iPhone gets online, message arrives.  In this moment, I would assume a notification would appear, but it does not! Only after I open Signal, I can see the new message. (I have notifications enabled of course).

https://debuglogs.org/ios/5.27.1/c43080fbc5db736bc8a07b6af37db60743536e9071093194edccf8b22bdaeeb9.zip

**Comment by SigniOS** [comment]
Zsign technical support:
Apple signature tool supporting ios15 and ios16
Support the removal of dylib dynamic library
Completely free, permanently free!
https://urlzs.com/44zCx

**Comment by T4cC0re** [comment]
Seems to be a dupe of #5072

**Comment by Cerberus0** [comment]
Perhaps it would be better to allow the stale bot to close this issue as a duplicate of #5072, unless someone can demonstrate how the issue experienced by the original reporter here is uniquely different from the one reported there.
